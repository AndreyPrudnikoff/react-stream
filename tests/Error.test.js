import React from 'react'
import { cleanup, fireEvent, render } from '@testing-library/react';
import renderer from 'react-test-renderer';
import { useHistory } from 'react-router-dom'
import { Provider, useDispatch } from "react-redux";
import store from '../src/store'
import Error from '../src/Error.js'

test('Links didn\'t change', () => {
    const component = renderer.create(
        <Provider store={store}><Error/></Provider>
    );
    let tree = component.toJSON();
    expect(tree).toMatchSnapshot();
});

afterEach(cleanup);

it('Error component exist', () => {
    const component = render(
        <Provider store={store}><Error/></Provider>
    );
});