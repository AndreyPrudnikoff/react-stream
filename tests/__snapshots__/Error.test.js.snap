// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Links didn't change 1`] = `
<div
  className="error"
  onClick={[Function]}
>
  <h2 />
</div>
`;
